zbuf/combiner.go

```diff
@@ -32,27 +32,33 @@ func CmpTimeReverse(a, b *zng.Record) bool {
 	return !CmpTimeForward(a, b)
 }
 
-type Combiner struct {
-	readers []Reader
-	hol     []*zng.Record
-	done    []bool
-	less    RecordCmpFn
-}
-
-// NewCombiner returns a Combiner that merges zbuf.Readers into
+// NewCombiner returns a ReaderCloser that merges zbuf.Readers into
 // a single Reader. If the ordering of the input readers matches
 // the direction specified, the output records will maintain
 // that order.
-func NewCombiner(readers []Reader, cmp RecordCmpFn) *Combiner {
-	return &Combiner{
+func NewCombiner(readers []Reader, cmp RecordCmpFn) ReadCloser {
+	if len(readers) == 1 {
+		// XXX
```

Comment:
Seems like a reasonable check, can this comment be removed?

Reply:
It certainly can. Done in 79d7a50f8d77978dcfb75a3beb8f1a7383403385.